lib/OpenQA/WebAPI/Auth/OpenID.pm

```diff
@@ -108,8 +108,10 @@ sub auth_response ($c) {
     };
 
     $csr->handle_server_response(
-        not_openid =>
-          sub () { $err_handler->('Failed to login', 'OpenID provider returned invalid data. Please retry again') },
+        not_openid => sub () {
+            my $txt = 'Please retry again. Likely, OpenID message is not considered a valid object. It is missing, at least, "openqa.mode" value';
```

Comment:
I assume we cannot give further suggestions to server operators reading this message? Should they get in touch with somebody? Just ignore it?

Reply:
The API doesn't seem to provide anything else here: [Net::OpenID::Consumer](https://metacpan.org/pod/Net::OpenID::Consumer).

Reply:
it is not very clear what cause the invalid args. Usually retry helps, right? 
As @okurz suggested on slack[0] maybe we can include some data from controller.

[0] https://suse.slack.com/archives/C02AJ1E568M/p1727358701821639?thread_ts=1727357210.018199&cid=C02AJ1E568M

Reply:
> I don't think the retry hint is useful in that logs as it's not directed to users

removed